codegen/src/main/java/org/web3j/codegen/AbiTypesGenerator.java

```diff
@@ -36,7 +36,7 @@ public static void main(String[] args) throws Exception {
         if (args.length == 1) {
             abiTypesGenerator.generate(args[0]);
         } else {
-            abiTypesGenerator.generate(System.getProperty("user.dir") + "/core/src/main/java/");
+            abiTypesGenerator.generate(System.getProperty("user.dir") + "/abi/src/main/java/");
```

Comment:
Not sure if this was intentional, but it looks like the datatype classes now live in the abi module.

Reply:
That was before I migrated the abi package into its own module.